From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Sun, 10 Mar 2019 19:30:12 +0100
Subject: prefs: Change default of show-fallback-app-menu

The app menu always was a GNOME-only thing, so after it was removed this
cycle, assuming that it is not displayed by the environment is a better
default.

https://gitlab.gnome.org/GNOME/mutter/issues/493

Origin: upstream, commit:574099449fd7cf78c226bdf9d3d5a414d4bd351
Applied-Upstream: 3.32.1
---
 src/core/prefs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/prefs.c b/src/core/prefs.c
index fdee5b2..3f0db8a 100644
--- a/src/core/prefs.c
+++ b/src/core/prefs.c
@@ -111,7 +111,7 @@ static gboolean resize_with_right_button = FALSE;
 static gboolean edge_tiling = FALSE;
 static gboolean force_fullscreen = TRUE;
 static gboolean auto_maximize = TRUE;
-static gboolean show_fallback_app_menu = FALSE;
+static gboolean show_fallback_app_menu = TRUE;
 
 static GDesktopVisualBellType visual_bell_type = G_DESKTOP_VISUAL_BELL_FULLSCREEN_FLASH;
 static MetaButtonLayout button_layout;
